libmultilabel/nn/model.py

```diff
@@ -152,14 +152,13 @@ def _shared_eval_epoch_end(self, step_outputs, split):
         self.eval_metric.reset()
         return metric_dict
 
-    def predict_step(self, batch, batch_idx, dataloader_idx):
+    def predict_step(self, batch, batch_idx):
```

Comment:
Could you please elaborate more on this issue?
Because I do not quite understand what the issue is and why this change resolves the issue.
Thanks.